code/modules/admin/fun_balloon.dm

```diff
@@ -83,3 +83,64 @@
 		PoolOrNew(/obj/effect/overlay/temp/sparkle, M)
 		M.forceMove(T)
 		M << "<span class='notice'>Pop!</span>"
+
+/obj/effect/station_crash
+	name = "station crash"
+	desc = "With no survivors!"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "syndballoon"
+	anchored = TRUE
+
+/obj/effect/station_crash/New()
+	for(var/S in SSshuttle.stationary)
+		var/obj/docking_port/stationary/SM = S
+		if(SM.id == "emergency_home")
+			var/new_dir = turn(SM.dir, 180)
+			SM.loc = get_ranged_target_turf(SM, new_dir, rand(3,15))
+			break
+	qdel(src)
+
+
+
+//meteor/asteroid shuttle slamming into the station
+
+/obj/effect/station_crash
+	name = "station crash"
+	desc = "With no survivors!"
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "syndballoon"
+	anchored = TRUE
+
+
+//Luxury Shuttle Blockers
+
+/obj/effect/forcefield/luxury_shuttle
+	var/threshhold = 500
+	var/list/approved_passengers = list()
+
+/obj/effect/forcefield/luxury_shuttle/CanPass(atom/movable/mover, turf/target, height=0)
+	if(mover in approved_passengers)
+		return 1
+
+	if(!isliving(mover)) //No stowaways
```

Comment:
They can actually just move past it because they're ghosts and incorporeal move doesn't give a shit; presumably this is for preventing closets/etc